WootricSDK/WootricSDK/Wootric.h

```diff
@@ -77,4 +78,9 @@
 
 + (void)setCustomValueForResurveyThrottle:(NSNumber *)resurveyThrottle visitorPercentage:(NSNumber *)visitorPercentage registeredPercentage:(NSNumber *)registeredPercentage dailyResponseCap:(NSNumber *)dailyResponseCap;
 
++ (void)setSendButtonBackgroundColor:(UIColor *)color;
++ (void)setNoThanksColor:(UIColor *)color;
++ (void)setSliderColor:(UIColor *)color;
++ (void)setSocialSharingColor:(UIColor *)color;
```

Comment:
I agree with @hachibu 

Just one note, the same as said above about the getters you overwrite for the declared properties, you don't need to explicitly declare them, just implement them in the .m file.